toolkits/global/packages/global-forms/view/fields/radios.hbs

```diff
@@ -0,0 +1,18 @@
+{{#if inputs}}
+	<fieldset class="c-forms__fieldset c-forms__field" {{#unless legend}} role="presentation"{{/unless}} {{> inputCommonAttributes}}>
```

Comment:
I don't think we can reuse `inputCommonAttributes` here because the fieldset then ends up with duplicate id to the the form field. I also spotted a similar issue on a `div`. If you look at the demo.html there are a few. 
eg in devtools console run `document.querySelectorAll('#textarea-example')` you will see two elements with the same id, a `div` and `textarea`. The textarea is a child element of the div

Reply:
Ah, good shout. I am still getting used to the way handlebars deals in inheritance (I prefer nunjucks). Plus, Firefox a11y inspector apparently doesn't identify duplicate `id`s. We don't actually need `id`s on the fields `<div>`s, I just put them there for scripting convenience. This should be fixed now.

Reply:
Maybe not for this PR directly but in future it would be good to use Pally or aXe on the component in its various states to see if theres any accessibility issues from a component level. We did a similar thing at GDS which worked well. Happy to talk you through it.